app/src/main/java/com/droidhats/campuscompass/models/NavigationRoute.kt

```diff
@@ -0,0 +1,22 @@
+package com.droidhats.campuscompass.models
+
+import com.google.android.gms.maps.model.LatLng
+
+class NavigationRoute (
+    var origin: Location?,
+    var destination: Location?,
+    var transportationMode : String?,
+    var polyLinePath : MutableList<List<LatLng>>,
+    var instructions : ArrayList<String>
+){
+
+    var routeTime: String? = null
+
+        enum class TransportationMethods(val string : String){
```

Comment:
Can we fix the spacing for before enum class to be consistent with the rest of class variables?